scripts/telemctl

```diff
@@ -75,11 +97,12 @@ telem_restart() {
 usage() {
   printf "$SCRIPT - Control actions for telemetry services\n"
   printf "\n"
```

Comment:
What was wrong with the formatting?  

If not using the format statements, echo does the same thing.

Reply:
The %-65s adds a lot of whitespace padding at the end of the line.
I would go with

format='%15s %s\n'
printf "$format" "stop" "Stops all running telemetry services"

Reply:
The comment formatting before change looks like a pyramid. That was the complain.

```
scripts/telemctl - Control actions for telemetry services

           stop   Stops all running telemetry services                             
          start   Starts all telemetry services                                    
        restart   Restarts all telemetry services                                  
         opt-in   Opts in to telemetry, and starts telemetry services              
        opt-out   Opts out of telemetry, and stops telemetry services 
```